.github/workflows/acceptance_tests_lite.yaml

```diff
@@ -0,0 +1,41 @@
+name: Terraform Provider Acceptance Tests Lite
+on:
+  #Every day at 08:00 UTC
+  schedule:
+    - cron: '0 8 * * *'
+  push:
+permissions:
+  # Permission for checking out code
+  contents: read
+jobs:
+  # Lite acceptance tests run on zerocloud provider only, which makes them really fast,
+  # but with limited coverage
+  acceptance_lite:
+    name: Acceptance Tests Lite (Terraform ${{ matrix.terraform-version }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
don't you want to be true?

Reply:
Not really. This "jobs.<job_id>.strategy.fail-fast" parameter if set to "true" (the default) will cancel all other runs in the matrix below. There are two runs with the two latest versions of Terraform. For our case it's worth it to pay the negligable cost of having the two E2E run, if the other fails. This may reveal an incompatibility between the Terraform 1.1.X and 1.2.X versions for our provider.